src/components/who.js

```diff
@@ -4,74 +4,117 @@ import styled from "styled-components"
 import Section from "../styles/Section"
 import Header from "../styles/Header"
 import Img from "gatsby-image"
+import shovel from "../images/shovel.svg"
 
-// Probably a better name
-const ContainerContainer = styled.div`
-  display: flex;
+// SVG gets inlined as base64. This is ok because it's only 3 kb
+const Mobile = styled.div`
+  @media (min-width: 1000px) {
+    display: none;
+  }
+`
+
+const Desktop = styled.div`
+  display: none;
+  @media (min-width: 1000px) {
+    display: grid;
+    grid-template-columns: 1fr 400px;
+    grid-column-gap: 40px;
+  }
 `
 
-const TextContainer = styled.div`
-  flex: 3;
-  padding-right: 40px;
+const ShovelBackground = styled(Section)`
+  background-image: url(${shovel});
+  background-position: 50% 100px;
+  background-size: 80%;
+  background-repeat: no-repeat;
+  @media (min-width: 1000px) {
```

Comment:
This is just a comment, not something that needs to be fixed before this gets merged, but we might want to experiment with this query, with respect to design. Not too keen on the look of it right now.